### DISCUSSION TITLE
Blocco aree per i genitori

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#273] [Idee e proposte]
Salve, sarebbe possibile creare nel menù preside una sezione dove bloccare ai genitori determinate funzioni?
E' molto utile negli scrutini sopratutto o a fine anno. 
Attendo riscontro.

**Comment by trinko** [comment]
Al momento non è previsto. Però tieni presente che:
- i genitori possono solo leggere dati, non inserirli;
- le richieste di colloquio sono bloccate automaticamente 30 giorni prima della fine dell'anno scolastico;
- gli esiti degli scrutini sono visibili solo dopo che viene attivata la pubblicazione;
- il resto che possono vedere i genitori sono informazioni che vedevano durante l'anno (perché nasconderle?) e la bacheca delle circolari e degli avvisi che è bene tenere sempre attivi.
- infine, in bacheca, possono vedere anche i programmi svolti che vengono caricati dai docenti.

In sintesi, che c'è da bloccare?

**Reply by manuelbarberi** [reply]
La sezione pagelle, voti, colloqui per l'anno... Magari alcune scuole vogliono bloccare la giustifica ecc.

**Reply by trinko** [reply]
> La sezione pagelle, voti, colloqui per l'anno... Magari alcune scuole vogliono bloccare la giustifica ecc.

I colloqui sono bloccati automaticamente. Perché non far più vedere i voti o le vecchie pagelle? 

Le giustificazioni forse sono l'unica cosa che si dovrebbe bloccare automaticamente alla fine dell'anno, per evitare che le assenze ingiustificate vengano alterate successivamente.

**Reply by manuelbarberi** [reply]
Lo so, ma in alcune scuole preferiscono fare così
